packages/protocol/thread/thread.ts

```diff
@@ -46,11 +46,13 @@ import {
 } from "@replayio/protocol";
 import groupBy from "lodash/groupBy";
 import uniqueId from "lodash/uniqueId";
+import { getAlternateSourceId, getPreferredSourceId } from "ui/reducers/sources";
```

Comment:
:hand: conceptually, this ends up dragging in a bunch of the "app" code.

Could we switch this up to injecting these via setter into `ThreadFront` as the app starts?

I know it's a bit silly, but it'd be nice to maintain that separation for now.

Reply:
That said: I've also just now realized that `ThreadFront` is used over in Brian's prototype, which has _no_ Redux store at all. So, that prototype is going to completely break because of this PR.

Reply:
> Could we switch this up to injecting these via setter into ThreadFront as the app starts?

Done.